imagehash_test.go

```diff
@@ -56,6 +56,18 @@ func TestNewImageHash(t *testing.T) {
 	}
 }
 
+func TestNil(t *testing.T) {
+	hash := NewImageHash(0, AHash)
+	dis, err := hash.Distance(nil)
+	expectedError := errors.New("other should not be nil")
+	if err != nil && err.Error() != expectedError.Error() {
+		t.Errorf("Expected err %s, actual %s", expectedError, err)
+	}
+	if dis != -1 {
+		t.Errorf("Distance is expected as %d but got %d", -1, dis)
+	}
```

Comment:
```suggestion
	dis, err := hash.Distance(nil)
	if err != nil && !errors.Is(err, errNoOther) {
		t.Errorf("Expected err %s, actual %s", expectedError, err)
	}
	if dis != -1 {
		t.Errorf("Distance is expected as %d but got %d", -1, dis)
	}
```

The `err != nil` can be omitted.

Reply:
Done!

Reply:
hmm errors.Is is not supported in old golang verison